http/handler.go

```diff
@@ -116,7 +116,7 @@ func stripPrefix(prefix, path string) (string, bool) {
 func parseRequest(r *http.Request, out interface{}) error {
 	// Limit the maximum number of bytes to MaxRequestSize to protect
 	// against an indefinite amount of data being read.
-	limit := &io.LimitedReader{R: r.Body, N: MaxRequestSize}
+	limit := &io.LimitedReader{R: r.Body, N: 2*MaxRequestSize}
```

Comment:
Body read is limited by MaxBytesReader in logical.go. Keeped by "hard security". Could be removed if needed.